Assets/Python/GreatPeople.py

```diff
@@ -1265,11 +1293,18 @@ def setup():
 		"Charles V", # 14th
 		"fJeanne d'Arc", # 15th
 		iRenaissance,
+		u"Louis de Bouron-Condé", # 17th
```

Comment:
There's a typo: it's [Louis de Bourbon-Condé](https://fr.wikipedia.org/wiki/Louis_II_de_Bourbon-Cond%C3%A9)